tests/foreman/cli/test_host_collection.py

```diff
@@ -679,35 +330,6 @@ def test_positive_copy_by_id(self):
         })
         self.assertEqual(result['name'], new_name)
 
-    @tier2
-    def test_positive_copy_by_name(self):
```

Comment:
Can you please confirm if we are not loosing coverage for host collection copy?

Reply:
There is another test that copy host collections by id few lines above. So I think this one is not necessary.